### PR TITLE
Install.rst: Install virtualenv with --user

### LINKED ISSUE #4017: Development_Setup.html: Tell users that they probably need to install virtualenv with sudo
If users install `virtualenv` for the first time through pip like so:

>Install virtualenv using pip :
>    $ pip3 install virtualenv

they most likely need to do that with sudo rights. We should add a note about that.

### PR DESCRIPTION
**Pull request by lilithxxx**
Updates documentation to tell user to Install virtualenv with --user and how to check whether its already installed.

Closes coala/coala#4017

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Review by newbazz — COMMENTED** [review]
You havent followed the commit guidelines ;)

**Comment by lilithxxx** [comment]
@newbazz What needs to be resolved?

**Review by newbazz — COMMENTED** [review]
Read the commit guidelines and follow them http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

File: Users/Install.rst [hunk 1]
```diff
@@ -90,13 +90,21 @@ Virtualenv is probably what you want to use during development.
...
-    $ pip3 install virtualenv
+    $ virtualenv --version
+
+If you see a version number, its already installed. If you do
+not, this can be done with pip3 easily:
+
+::
+
+    $ pip3 install --user virtualenv
+
```
Comment (kriti21) [inline Users/Install.rst]: Only one blank line is needed here. Please delete the extra empty line.

**Review by kriti21 — CHANGES_REQUESTED** [review]

File: Users/Install.rst [hunk 2]
```diff
@@ -90,13 +90,20 @@ Virtualenv is probably what you want to use during development.
...
+check if it's already installed by typing:
 
 ::
 
-    $ pip3 install virtualenv
+    $ virtualenv --version
+
+If you see a version number, its already installed. If you do
+not, this can be done with pip3 easily:
+
```
Comment (pareksha) [inline Users/Install.rst]: You can improve this by mentioning the output that the user will see if ``virtualenv`` is not installed.

Comment (lilithxxx) [reply Users/Install.rst]: I have already mentioned that the user will not see a version number if `virtualenv` is not installed and I think the output message would be pretty obvious to know that its not installed yet

File: Users/Install.rst [hunk 3]
```diff
@@ -90,13 +90,20 @@ Virtualenv is probably what you want to use during development.
...
+have this installed as ``virtualenv`` or ``pyvenv``. You can
+check if it's already installed by typing:
 
 ::
 
-    $ pip3 install virtualenv
+    $ virtualenv --version
+
+If you see a version number, its already installed. If you do
+not, this can be done with pip3 easily:
```
Comment (newbazz) [inline Users/Install.rst]: You need to explain the use of `--user` flag.

**Review by jayvdb — CHANGES_REQUESTED** [review]
pls squash your two commits into a single commit.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

File: Users/Install.rst [hunk 4]
```diff
@@ -132,13 +132,22 @@ steps given below.
...
+
+::
+
+    $ pip3 install --user virtualenv
+
+The ```--user``` tag is required to install it with root access.
```
Comment (jayvdb) [inline Users/Install.rst]: with -> without

**Review by jayvdb — CHANGES_REQUESTED** [review]
One small fix, and please squash your commits, and rebase.

**Pull request closed without merging**